docs/releases/1.9.txt

```diff
@@ -239,6 +239,9 @@ Minor features
 * Added the :class:`~django.contrib.postgres.functions.TransactionNow` database
   function.
 
+* Fixed :class:`~django.contrib.postgres.forms.HStoreField` which has never
+  worked.
+
```

Comment:
Since it will be backported to 1.8, this is in the wrong release notes anyways :wink: Can you add it to `1.8.4.txt` please.